libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -115,6 +114,7 @@ impl RestResourceProvider for BiomeRestResourceManager {
 
         #[cfg(all(
             feature = "biome-credentials",
+            feature = "biome-refresh-tokens",
```

Comment:
Are the rest of the routes here guarded by the biome-refresh-tokens feature? It doesn't seem like they are since this includes the credentials list route and the users routes.

Reply:
Re arranged commits so this gets added when the refresh token store gets added to the login route